src/package/README.Windows.pkg

```diff
@@ -0,0 +1,51 @@
+EverParse: verified parsing for binary data formats
```

Comment:
`src/package/README.*.pkg` are for the binary package, whereas `README.md` is for the GitHub clone, for those who want to build from source. The problem is that invocation of EverParse is different in the two cases, because:
* the `everparse.bat` or `everparse.sh` designed for the package will not work from the GitHub clone since it assumes locations for F* and KReMLin that are valid only for the package (and no, those scripts are not designed for users to supersede `FSTAR_HOME` etc.)
* users of the binary package could run `3d` directly from the package, but they would need to manually set up their environment with `FSTAR_HOME` etc., which would defeat the purpose of a standalone binary package